advanced networking/deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+echo "This script will deploy resouces which will enable you to work securely in a private virtual network".
+echo "Enter the Resource Group name:" &&
+read resourceGroupName &&
+echo "Enter the managed identity name:" &&
+read idName &&
+echo "Enter a password for connecting to vm:" &&
+read password &&
+# 1. Create resource group
+az group create -n $resourceGroupName -l southcentralus
+
+# 2. Create user assigned MI for running scripts in ARM templates
+identity=$(az identity create -g $resourceGroupName -n $idName --query id -o tsv)
+
+# 3. Assign contributor role for the MI on the RG
```

Comment:
>Assign contributor [](start = 5, length = 18)

Should this role assignment and/or MI be cleaned up after the script runs?  Is it only needed for running the deployment?